apf/apfMixedNumbering.cc

```diff
@@ -228,8 +227,9 @@ static int number_ghost(
 static void globalize(
     int dofs,
     std::vector<Numbering*> const& owned,
-    std::vector<GlobalNumbering*>& global) {
-  long start = PCU_Exscan_Long(long(dofs));
+    std::vector<GlobalNumbering*>& global,
+    pcu::PCU *PCUObj) {
+  long start = PCUObj->Exscan(long(dofs));
```

Comment:
PCUObj->Exscan<long>(dofs);

Reply:
```suggestion
  long start = PCUObj->Exscan<long>(dofs);
```

Reply:
Yup, Cameron is correct. I missed the type specification.